### ISSUE TITLE
SOD - Rune abilities aren't showing on enemies

### ISSUE THREAD

**Issue by rukk1** [casualshammy/NameplateAuras#227]
Heya, having issues having abilities like druid's rune Berserk showing up on the nameplate.  I'm having the same issue on just about every other addon, I assume something about how runes fire is different.  Thanks

**Comment by Theguyxd** [comment]
Have you manually added it?

**Comment by rukk1** [comment]
Yes, using a mixture of either specific spell ID or without spell ID and confirming the addon's tooltip has the correct spell ID.  Weird thing is I see the aura gain in MikScrollingBattleText but not on this addon.

**Comment by Theguyxd** [comment]
I added "Between the Eyes" rune for rogues. It seems to work just fine. All I did was type the name. You got the correct letters in the ability name capitalized?

**Comment by rukk1** [comment]
> I added "Between the Eyes" rune for rogues. It seems to work just fine. All I did was type the name. You got the correct letters in the ability name capitalized?

Yep, as I said the spell in the list has the correct spell ID, are you able to get things like Berserk or Pain Suppression working?  Seems like it has issue with phase 2 runes.

**Comment by casualshammy** [comment]
Hi,
Unfortunately I don't play classic and don't have high level druid so I cannot investigate this issue.
If rune spell is in the list but don't display on nameplates (and you're sure filters are ok) then there is some unique 'method' of detecting such runes. Quick googling and looking in source code of some addons didn't help much. >_<

**Comment by rukk1** [comment]
What other details or troubleshooting and I provide/do to help?

**Comment by casualshammy** [comment]
> What other details or troubleshooting and I provide/do to help?

Hmm, well, try to target druid with active `Berserk` buff and then enter this command in chat:
```
/run for i=1,40 do local n,_,_,_,_,_,_,_,_,s=UnitAura("target", i);if(n)then print(n,s);end;end;for i=1,40 do local n,_,_,_,_,_,_,_,_,s=UnitBuff("target", i);if(n)then print(n,s);end;end
```

You'll see a list of active auras/buffs with corresponding IDs in your chat like this:
![image](https://github.com/casualshammy/NameplateAuras/assets/4965168/210a136e-0568-43b9-9912-8c2e33a6519a)

Please copy or make screenshot of this list. This will show if `Berserk` buff is detectable in principle by calling `UnitBuff`/`UnitAura` functions.

**Comment by rukk1** [comment]
> `/run for i=1,40 do local n,_,_,_,_,_,_,_,_,s=UnitAura("target", i);if(n)then print(n,s);end;end;for i=1,40 do local n,_,_,_,_,_,_,_,_,s=UnitBuff("target", i);if(n)then print(n,s);end;end`

Heya, been trying to. But having trouble finding a way to test it.  Duels I think the buffs are treated differently?
And just to be clear it's all rune abilities not just Berserk, and since phase 2 things have been weird.  Whatever changed broke addons like ClassicAuraDurations.
